src/Symfony/Component/Console/Style/SymfonyStyle.php

```diff
@@ -46,7 +47,7 @@ class SymfonyStyle extends OutputStyle
     public function __construct(InputInterface $input, OutputInterface $output)
     {
         $this->input = $input;
-        $this->bufferedOutput = new BufferedOutput($output->getVerbosity(), false, clone $output->getFormatter());
+        $this->bufferedOutput = new TrimmedBufferOutput($output->getVerbosity(), false, unserialize(serialize($output->getFormatter())), \DIRECTORY_SEPARATOR === '\\' ? 4 : 2);
```

Comment:
Can this be reverted to use `clone $output->getFormatter()` now that the bug is fixed ?

Reply:
done